src/components/MentorsHome/MentorsHome.js

```diff
@@ -25,217 +21,68 @@ const styles = theme => ({
-    if (this.props.authUser) {
-      this.mentorState();
-    }
-  };
-  componentDidUpdate(prevProps) {
-    if (this.props !== prevProps) {
-      this.mentorState();
-    }
-  }
   render() {
     const { classes } = this.props;
-    const {
-      mentorState,
-      openSnackbarSaved,
-      openSnackbarError,
-      sectionError,
-      successMsg
-    } = this.state;
+
     return (
       <div className={classes.root}>
         <div className={classes.pageTitle}>
           <Typography variant="h6" gutterBottom>
-            Thank you for being a mentor
+            Welcome to FLAD Mentorship
           </Typography>
         </div>
-        <Card className={classes.card}>
-          <CardContent>
-            <Typography variant="body1" gutterBottom>
-              {mentorState ? HAVE_SET_STATE : HAVENT_SET_STATE}
+        <Grid container>
+          <Grid item xs={12} sm={6} md={6} lg={6}>
+            <img
+              src={logo_original}
+              alt="Let's talk about business"
+              className={classes.principalLogo}
+            />
+          </Grid>
+          <Grid item xs={12} sm={6} md={6} lg={6}>
+            <Typography variant="h6" color="primary" gutterBottom>
+              Thank you for being a mentor!
             </Typography>
-            <div>
-              <TextField
-                id="mentorState"
-                name="mentorState"
-                multiline
-                rows="5"
-                label="Your message (200 characters max)"
-                value={mentorState}
-                onChange={this.handleChange}
-                className={classes.textField}
-                inputProps={{ maxLength: 200 }}
-              />
-            </div>{" "}
+            <Typography variant="body1">
+              Mentorship is incredibly important because it provides
+              participating mentees with valuable insight to assist and guide
+              them in creating, growing, and strengthening their businesses, and
+              achieve success.
+            </Typography>
+            <br />
+            <Typography variant="body1">
+              {" "}
+              As a mentor, you will be able to share your expertice with the
+              community and be involve with a wide business network!.
+            </Typography>
+            <br />
+            <Typography variant="body1">
+              {" "}
+              You can share a message with our community anytime clicking the
+              button bellow.
+            </Typography>
+            <br />
             <div>
               <Button
                 variant="contained"
                 color="primary"
-                type="submit"
-                fullWidth
+                component={Link}
+                to="/mentorsmessage"
                 className={classes.button}
-                onClick={this.handleSubmit}
               >
-                Share
+                Write a message
```

Comment:
There is a possibility to have the message input here, instead of creating a new route?